src/app/core/services/split-expense.service.ts

```
@@ -415,4 +417,104 @@
+      );
+
+      totalRoundedAmount += roundedAmount;
+    });
+
+    const difference = parseFloat((roundedTargetAmount - totalRoundedAmount).toFixed(precision + 1));
+
+    if (Math.abs(difference) > threshold) {
+      const lastIndex = splitExpensesFormArray.length - 1;
+      const lastControl = splitExpensesFormArray.at(lastIndex);
+      const lastAmount = (lastControl.get('amount')?.value as number) || 0;
+      const adjustedAmount = parseFloat((lastAmount + difference).toFixed(precision));
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)